Tests/Form/FieldTypeHandler/CheckboxTest.php

```diff
@@ -5,62 +5,235 @@
 {
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject
+     */
+    protected $fieldHelper;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject
+     */
+    protected $formBuilder;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject
+     */
+    protected $content;
+
+    /**
+     * @var array
+     */
+    protected $fieldDefinitionParameters;
+
+    protected function setUp()
+    {
+        $this->fieldHelper = $this->getMockBuilder(FieldHelper::class)
+            ->disableOriginalConstructor()
+            ->setMethods(array('isFieldEmpty'))
+            ->getMock();
+
+        $this->formBuilder = $this->getMockBuilder(FormBuilder::class)
+            ->disableOriginalConstructor()
+            ->setMethods(array('add'))
+            ->getMock();
+
+        $this->content = $this->getMockBuilder('eZ\Publish\API\Repository\Values\Content\Content')
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->fieldDefinitionParameters = array(
+            'id' => 'id',
+            'identifier' => 'identifier',
+            'isRequired' => true,
+            'defaultValue' => new CheckboxValue(true),
+            'descriptions' => array('fre-FR' => 'fre-FR'),
+            'names' => array('fre-FR' => 'fre-FR'),
+            'validatorConfiguration' => array(),
+        );
+    }
+
     public function testAssertInstanceOfFieldTypeHandler()
     {
-        $checkbox = new Checkbox();
+        $checkboxHandler = new Checkbox($this->fieldHelper);
 
-        $this->assertInstanceOf(FieldTypeHandler::class, $checkbox);
+        $this->assertInstanceOf(FieldTypeHandler::class, $checkboxHandler);
     }
 
     public function testConvertFieldValueToForm()
     {
-        $checkbox = new Checkbox();
-        $checkboxValue = new CheckboxValue\Value(true);
+        $checkboxHandler = new Checkbox($this->fieldHelper);
+        $checkboxValue = new CheckboxValue(true);
 
-        $returnedBool = $checkbox->convertFieldValueToForm($checkboxValue);
+        $returnedBool = $checkboxHandler->convertFieldValueToForm($checkboxValue);
 
         $this->assertTrue($returnedBool);
     }
 
     public function testConvertFieldValueFromForm()
     {
-        $checkbox = new Checkbox();
-        $checkboxValue = new CheckboxValue\Value(true);
+        $checkboxHandler = new Checkbox($this->fieldHelper);
+        $checkboxValue = new CheckboxValue(true);
 
-        $returnedBool = $checkbox->convertFieldValueFromForm(true);
+        $returnedBool = $checkboxHandler->convertFieldValueFromForm(true);
 
         $this->assertEquals($checkboxValue, $returnedBool);
     }
 
+
     public function testBuildFieldCreateForm()
     {
-        $formBuilder = $this->getMockBuilder(FormBuilder::class)
-            ->disableOriginalConstructor()
-            ->setMethods(array('add'))
-            ->getMock();
+        $formBuilder = clone($this->formBuilder);
```

Comment:
Why cloning? `setUp()` should be automatically called before each test, so this should not be needed.